recipe/meta.yaml

```diff
@@ -108,6 +108,7 @@ outputs:
         # run-like
         - alembic >=1.13.0
         - astropy-base >=7.0.1, <8
+        - astroplan >=0.10.1
```

Comment:
I think it being in rubin-env means that the line can be removed from rubin-env-rsp.